tools/status/report.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+set -e
+
+BUCKET=airbyte-status
+
+CONNECTOR=$1
+REPOSITORY=$2
+RUN_ID=$3
+OUTCOME=$4
+
+BUCKET_WRITE_ROOT=/tmp/bucket_write_root
+LAST_TEN_ROOT=/tmp/last_ten_root
+SUMMARY_WRITE_ROOT=/tmp/summary_write_root
+
+export AWS_PAGER=""
+
+# write log of job run to s3
+rm -r $BUCKET_WRITE_ROOT || true
+mkdir -p $BUCKET_WRITE_ROOT
+cd $BUCKET_WRITE_ROOT
+mkdir -p tests/history/"$CONNECTOR"
+LINK=https://github.com/$REPOSITORY/actions/runs/$RUN_ID
+TIMESTAMP="$(date +%s)"
+echo "{ \"link\": \"$LINK\", \"outcome\": \"$OUTCOME\" }" > tests/history/"$CONNECTOR"/"$TIMESTAMP".json
+aws s3 sync "$BUCKET_WRITE_ROOT"/tests/history/"$CONNECTOR"/ s3://"$BUCKET"/tests/history/"$CONNECTOR"/
+
+# pull the logs for the latest ten jobs for this connector
+LAST_TEN_FILES=$(aws s3api list-objects-v2 --bucket "$BUCKET"  \
+  --query "reverse(sort_by(Contents[?contains(Key, \`tests/history/$CONNECTOR\`)], &LastModified))[:10].Key" \
+  --output=text)
+
+rm -r $LAST_TEN_ROOT || true
+mkdir -p $LAST_TEN_ROOT
+
+for file in $LAST_TEN_FILES; do
+  aws s3 cp s3://"$BUCKET"/"$file" $LAST_TEN_ROOT/
+done
+
+# build and push tests/CONNECTOR/badge.json and tests/CONNECTOR/index.html
+HTML_TABLE_ROWS=""
+
+successes=0
+failures=0
+
+while IFS= read -r file; do
+  line=$(cat "$LAST_TEN_ROOT/$file")
+  outcome=$(echo "$line" | jq -r '.outcome')
+  if [ "$outcome" = "success" ]; then
+    successes=$((successes+1))
+    outcome_output="<span style=\"color:green;\">&#10004; $outcome</span>"
+  else
+    failures=$((failures+1))
+    outcome_output="<span style=\"color:red;\">&#10008; $outcome</span>"
+  fi
+  LINK=$(echo "$line" | jq -r '.link')
+  HTML_TABLE_ROWS="<tr><td>$(date -d @"$(echo "$file" | cut -f 1 -d '.')")</td><td>$outcome_output</td><td><a href=\"$LINK\">$LINK</a></td></tr>$HTML_TABLE_ROWS"
```

Comment:
`date -r` works on Mac but doesn't work on Github Actions. `date -d @` works on Github Actions but not on Mac. This is the only thing that prevents this script from being manually run locally.

Reply:
if that's the case can you just do a switch based on how the OS? it's ugly but i think it's important that it runs in both places.